torchx/lib/torchx/backend.ex

```diff
@@ -1069,6 +1113,29 @@ defmodule Torchx.Backend do
     |> to_nx(out)
   end
 
+  def unfold_windows(%T{} = tensor, padding, pad_constant, window_dims_tuple, strides) do
+    unfold_windows(from_nx(tensor), padding, pad_constant, window_dims_tuple, strides)
+  end
+
+  def unfold_windows(tensor, padding, pad_constant, window_dims_tuple, strides) do
+    padding =
+      padding
+      |> Enum.map(fn {a, b} -> [a, b] end)
+      |> Enum.reverse()
+      |> List.flatten()
```

Comment:
You can probably do this much more efficiently with a single reduce:

```suggestion
      Enum.reduce(padding, [], fn {a, b}, acc -> [a, b | acc] end)
```

There may be other usages here. It may be worth extracting to a function too. Like flatten_padding. :D